### ISSUE TITLE
Undefined property error in 0.2.6 

### ISSUE THREAD

**Issue by mallardduck** [mnapoli/imapi#9]
It seems that after the merge of #6 a breaking change was introduced that causes issues in the _current_ usage of Queries. The error as it is reads off:

```
[2018-01-30 12:14:01] production.ERROR: Undefined property: Imapi\Query\Query::$flags {"exception":"[object] (ErrorException(code: 0): Undefined property: Imapi\\Query\\Query::$flags at /home/srv/laravel/utilities/vendor/mnapoli/imapi/src/Query/Query.php:54)
[stacktrace]
```

My gut feeling is that this is being caused because the $flags class property isn't being explicitly defined, nor set to default to an array. I will submit a PR correcting for that issue.

**Comment by mnapoli** [comment]
Thanks for the detailed report and thanks for that:

> I will submit a PR correcting for that issue.

:)

**Comment by mallardduck** [comment]
Woot! This is resolved with #10 being merged in.

:man_dancing:
